README.md

```diff
@@ -52,6 +52,43 @@ FeatureGateMyFeatureName = newFeatureGate("MyFeatureName").
 			mustRegister()
 ```
 
+### defining tests
+Tests are logically associated with FeatureGates.
+When adding any FeatureGated functionality a new test file is required.
+The test files are located in `<group>/<version>/tests/<crd-name>/FeatureGate.yaml`
+```
+route
+  v1
+    tests
+      routes.route.openshift.io
+        AAA_ungated.yaml
+        ExternalRouteCertificate.yaml
+```
+
+Here's an `AAA_ungated.yaml` example
+```yaml
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Route"
```

Comment:
I'm a fan of proper punctuation for English prose.  However, the double-quotes around "Route" are conspicuous, especially as none of the other values are quoted in the yaml—are the double-quotes actually needed?
````suggestion
Here's an `AAA_ungated.yaml` example:
```yaml
apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this.
name: Route
````